src/server/services/procedures/new-york-public-library/new-york-public-library.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const ApiConsumer = require('../utils/api-consumer');
+const {NewYorkPublicLibraryKey} = require('../utils/api-key');
+const NYPL = new ApiConsumer('NewYorkPublicLibrary', 'http://api.repo.nypl.org/api/v1', {cache: {ttl: 5*60}});
+ApiConsumer.setRequiredApiKey(NYPL, NewYorkPublicLibraryKey);
+
+function listify(item) {
+    return item === undefined ? [] : item instanceof Array ? item : [item];
+}
+
+/**
+ * Searches the NYPL db an returns up to count ids denoting matching items.
+ * 
+ * @param {String} term Term to search for
+ * @param {BoundedNumber<1>} per_page Maximum number of items to return for any given page
```

Comment:
Ah, right, I've been in rust land too long - I'll change to cammelcase and give default.

Reply:
Ok, I added a default and updated jsdoc with the `=` thing ({`BoundedNumber<1>=}` and it seems to work in NetsBlox, but it doesn't show any indication of what the default value is - just looks empty. Is that expected?

Reply:
Yeah, unfortunately that is the case now and it needs to be added to the argument info manually. It would probably be good to support the JSDoc version listed [here](https://jsdoc.app/tags-param.html) rather than the Google Closure Compiler version (or maybe along with that version).